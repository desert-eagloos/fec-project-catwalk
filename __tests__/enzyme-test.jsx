import React from 'react';
import expect from 'expect';
import Enzyme, { shallow } from 'enzyme';

import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

import Welcome from '../client/src/components/App';

Enzyme.configure({ adapter: new Adapter() });

describe('Welcome (Snapshot)', () => {
  it('Welcome renders PROJECT CATWALK', () => {
    const welcome = shallow(<Welcome />);
    expect(welcome.find('div').text()).toEqual('Hello World!<Overview /><QARoot />');
  });
});
