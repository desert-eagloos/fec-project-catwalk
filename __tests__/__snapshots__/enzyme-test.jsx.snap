// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome (Snapshot) Welcome renders PROJECT CATWALK 1`] = `ShallowWrapper {}`;
